### PR TITLE
dotool: 1.3 -> 1.5

### PR DESCRIPTION
**Pull request by idlip**
## Description of changes

Changes: https://git.sr.ht/~geb/dotool/tree/master/item/CHANGELOG.md

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by idlip** [comment]
Hi, I have included manual page. Hope way of just copying is fine.

I was wondering how to use `dotool` without root, even tho I'm in `input` group, and made this PR to bump version.

Can anyone let me know if we need module or how to make it work without root?

Thank you!

File: pkgs/tools/dotool/default.nix [hunk 1]
```diff
@@ -27,13 +28,15 @@ buildGoModule rec {
...
+  nativeBuildInputs = [ pkg-config scdoc ];
 
   ldflags = [ "-s" "-w" "-X main.Version=${version}" ];
 
   postInstall = ''
-    mkdir -p $out/bin
+    mkdir -p $out/{bin,share/man/man1}
     cp ./dotoold ./dotoolc $out/bin
+    scdoc < doc/dotool.1.scd > dotool.1
+    cp ./dotool.1 $out/share/man/man1/
```
Comment (NickCao) [inline pkgs/tools/dotool/default.nix:39]: Please use `installManPage` helper function from `installShellFiles`

Comment (idlip) [reply pkgs/tools/dotool/default.nix]: Thank you, updated it!

**Comment by idlip** [comment]
For anyone wanting to make it work for user, make sure user is in `uinput` group actually.

And enable;
`hardware.uinput.enable = true;`

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 326320` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>dotool</li>
  </ul>
</details>

**Review by smancill — APPROVED** [review]

**Pull request merged**